server/routes/createGoal/createGoalController.ts

```diff
@@ -54,6 +54,36 @@ export default class CreateGoalController {
     return res.render('pages/goal/create/index', { ...view.renderArgs })
   }
 
+  // TODO: RR-734 - Create controller and view classes new create goal journey
```

Comment:
The existing `CreateGoalController` is going to get way too big (it's too big already to be honest); so we don't want these new methods etc in this class.
Can you create a new controller class please, perhaps called `CreateGoalsController` or similar, and put your 2 new methods in it please?